config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -69,6 +69,6 @@ periodics:
       - "--acsengine-win-binaries"
       - "--acsengine-agentpoolcount=3"
       - "--test_args=--node-os-distro=windows --ginkgo.focus=\\[Conformance\\]|\\[NodeConformance\\]|\\[sig-windows\\]|\\[sig-apps\\].CronJob|\\[sig-api-machinery\\].ResourceQuota|\\[sig-scheduling\\].SchedulerPreemption --ginkgo.skip=\\[LinuxOnly\\]|\\[k8s.io\\].Pods.*should.cap.back-off.at.MaxContainerBackOff.\\[Slow\\]\\[NodeConformance\\]|\\[k8s.io\\].Pods.*should.have.their.auto-restart.back-off.timer.reset.on.image.update.\\[Slow\\]\\[NodeConformance\\]"
-      - "--timeout=450m"
+      - "--timeout=460m"
```

Comment:
I would add 20 to account for flakes.

Reply:
You should also add 20 mins to line 51.

L51 is the timeout for the job, L72 is the timeout for kubetest. The job timeout should be 20 mins greater than the one for kubetest.

Reply:
I changed both. Thanks @adelina-t . Its ready for review!